sdk/eventgrid/azure-messaging-eventgrid/migration-guide.md

```diff
@@ -0,0 +1,141 @@
+# Guide for migrating to `azure-messaging-eventgrid` from `microsoft-azure-eventgrid`
+
+This guide is intended to assist in the migration to `azure-messaging-eventgrid` from `microsoft-azure-eventgrid`. It will focus on side-by-side comparisons for similar operations between the two packages.
+
+We assume that you are familiar with the old SDK `microsoft-azure-eventgrid`. If not, please refer to the new SDK README for [azure-messaging-eventgrid][README] directly rather than this migration guide.
+
+## Table of contents
+
+- [Guide for migrating to `azure-messaging-eventgrid` from `microsoft-azure-eventgrid`](#guide-for-migrating-to-azure-messaging-eventgrid-from-microsoft-azure-eventgrid)
+    - [Table of contents](#table-of-contents)
+    - [Migration benefits](#migration-benefits)
+        - [Cross Service SDK improvements](#cross-service-sdk-improvements)
+    - [Important changes](#important-changes)
+        - [Group id, artifact id, and package names](#group-id-artifact-id-and-package-names)
+        - [Instantiating clients](#instantiating-clients)
+        - [Send events to an Event Grid Topic](#send-events-to-an-event-grid-topic)
+        - [Deserialize events and data](#deserialize-events-and-data)
+    - [Additional samples](#additional-samples)
+
+## Migration benefits
```

Comment:
Should this also highlight that the new library supports CloudEvent while the track 1 lib didn't?

Reply:
This part is usually high level general benefits and is not related to this SDK.
For instances, [Event Hubs migration guideline](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/eventhubs/azure-messaging-eventhubs/migration-guide.md#migration-benefits), [Service Bus](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/servicebus/azure-messaging-servicebus/migration-guide.md#migration-benefits), and [Key Vaule Secrets](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/azure-security-keyvault-secrets/migration_guide.md#migration-benefits)

Reply:
The template has a section for new features where this can be included.